app/utilities/schema.py

```diff
@@ -97,7 +97,7 @@ def _schema_exists(language_code: str, schema_name: str) -> bool:
     )
 
 
-def get_allowed_languages(schema_name: str | None, launch_language: str):
+def get_allowed_languages(schema_name: str | None, launch_language: str) -> list:
```

Comment:
This one can be more specific as the languages are all strings so `list[str]` works

Reply:
Yep, fixed that, thanks 👍